exercises/svm/ex_rnw/sol_svm_kernel_definition.Rnw

```diff
@@ -0,0 +1,99 @@
+\begin{enumerate}
+	\item 
+
+		The function is symmetric, as we can prove in the following way:
+
+		\begin{equation}
+			\kxxt  =  \xv^T \xtil  = \xtil^T \xv  =  k(\tilde \xv, \xv)
+		\end{equation}
+
+		To check if the kernel gram matrix is positive definite, we will remember the definition of $\xv$
+
+		\begin{equation}
+			\xv = \xdat \in \R^{p \times n}
+		\end{equation}
+
+		Taking that into account, the kernel gramn matrix is defined as :
+
+		\begin{equation}
+			\begin{aligned}
+				\mathbf{K}  &= \left( 
+						\begin{matrix}
+							\scp{\xi[1]}{\xi[1]} & \scp{\xi[1]}{\xi[2]} & \cdots & \scp{\xi[1]}{\xi[n]} \\
+							\scp{\xi[2]}{\xi[1]} & \scp{\xi[2]}{\xi[2]} & \cdots & \scp{\xi[2]}{\xi[n]} \\
+							\vdots & \vdots & \ddots & \vdots \\
+							\scp{\xi[n]}{\xi[1]} & \scp{\xi[n]}{\xi[2]} & \cdots & \scp{\xi[n]}{\xi[n]}
+						\end{matrix}
+					\right) \\
+				&= \xdat^T \xdat \\
+				&= \xv^T \xv \in \R^{n \times n}
+			\end{aligned}
+		\end{equation}
+
+		\begin{equation}
+			\begin{aligned}
+				\mathbf{a}^T \mathbf{K} \mathbf{a} & = \mathbf{a}^T \xv^T \xv \mathbf{a} \\
+				&= (\underbrace{\mathbf{a} \xv}_\mathbf{z})^T (\mathbf{a} \xv) \\
+				&= \mathbf{z}^T \mathbf{z} \\
+				&= \| \mathbf{z} \|_2^2 > 0
+			\end{aligned}
+		\end{equation}
+
+		We can conclude that this function is a kernel.
+
+	\item
+		
+		The function is not a kernel,
+		as we can prove that the kernel gram matrix is not positive definite. We will use $\xi[1]  = \frac{\pi}{2}$ and $a = 1$ :
+
+		\begin{equation}
+			\begin{aligned}
+				\mathbf{K} = k(\xi[1], \xi[1]) &= \cos\left(\frac{\pi}{2} + \frac{\pi}{2}\right)  =-1 \\
+				\mathbf{a}^T \mathbf{K} \mathbf{a} & = 1 \cdot -1 \cdot 1 = -1 < 0
```

Comment:
better without the transposed sign and with non-bold a since it is a scalar

Reply:
Done